src/platform/packages/shared/kbn-scout/src/playwright/utils/axe.test.ts

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the "Elastic License
+ * 2.0", the "GNU Affero General Public License v3.0 only", and the "Server Side
+ * Public License v 1"; you may not use this file except in compliance with, at
+ * your election, the "Elastic License 2.0", the "GNU Affero General Public
+ * License v3.0 only", or the "Server Side Public License, v 1".
+ */
+
+import { runA11yScan } from './axe';
+import { isValidUTCDate, formatTime, getPlaywrightGrepTag } from './runner_utils';
+
+describe('runA11yScan', () => {
+  test('returns violations array (empty for basic accessible markup)', async ({ page }) => {
```

Comment:
please move it to other "internal" e2e tests under `src/platform/packages/shared/kbn-scout/test/scout/ui`